h/static/scripts/streamer.js

```diff
@@ -27,33 +21,29 @@ var socket;
  * @return The push notification service client.
  */
 // @ngInject
-function connect($rootScope, annotationMapper, groups, session, settings) {
-  // The public interface of the streamer. Returned from connect.
-  var controls = {
-    setConfig: setConfig
-  };
+function Streamer($rootScope, annotationMapper, groups, session, settings) {
+
+  // The randomly generated session UUID
+  var clientId = uuid.v4();
```

Comment:
`clientId` is not exported here but several other places in the function assume that it is accessible as a property on the class instance.